README.md

```diff
@@ -1,13 +1,12 @@
 # Project Mongo API
 
-Replace this readme with your own information about your project.
-
-Start by briefly describing the assignment in a sentence or two. Keep it short and to the point.
+Project about how to use Mongodb to store data, and how to query that data from API. 
 
 ## The problem
 
-Describe how you approached to problem, and what tools and techniques you used to solve it. How did you plan? What technologies did you use? If you had more time, what would be next?
+A basic Netflix API built with Heroku and Express where the data is stored with Mongodb.
+I started from last projects with endpoints. I spend some time zooming out and trying to understand how everything belongs.
 
 ## View it live
+https://week-18-mongodb.herokuapp.com/
```

Comment:
Unfortunately I can't open the link, I get the Application Error-message. Maybe it's something with the Atlas cloud?